src/main/java/com/minecolonies/api/colony/buildings/registry/BuildingEntry.java

```diff
@@ -51,9 +52,9 @@ public static final class Builder
          * @param buildingBlock The block.
          * @return The builder.
          */
-        public Builder setBuildingBlock(final AbstractBlockHut<?> buildingBlock)
+        public Builder setBuildingBlock(final DeferredBlock<? extends AbstractBlockHut<?>> buildingBlock)
```

Comment:
This doesn't make a lot of sense, why not .get already then?

Reply:
less changes + less code? can change the param to supplier<...> if you want